content/docs/zero/install.mdx

```diff
@@ -18,39 +18,6 @@ If you haven't signed up for Pomerium Zero, you can [**create an account**](http
 :::
 
 <Tabs>
-<TabItem value="linux" label="Linux">
```

Comment:
The order of these tabs in the onboarding flow is still Linux, Kubernetes, Docker, Manual. Would it make sense to keep the same order in the docs?

Reply:
That makes sense. I'll update to so it's consistent with the onboarding flow.